Gems/Atom/Feature/Common/Code/Include/Atom/Feature/Utils/FrameCaptureBus.h

```diff
@@ -79,6 +80,19 @@ namespace AZ
             virtual uint32_t CapturePassAttachmentWithCallback(const AZStd::vector<AZStd::string>& passHierarchy, const AZStd::string& slotName
                 , RPI::AttachmentReadback::CallbackFunction callback, RPI::PassAttachmentReadbackOption option) = 0;
 
+            //! Compare 2 screenshots files and give scores (using root mean square RMS) for the difference.
+            //! @param filePathA the full path of screenshot A
+            //! @param filePathB the full path of screenshot B
+            //! @param diffScore [out] the RMS result
+            //! @param filteredDiff [out] an alternate RMS value calculated after removing any diffs less than @minDiffFilter.
+            //! @param minDiffFilter diff values less than this will be filtered out before calculating @filteredDiff.
+            //! @return if the comparison is successfully performed
+            virtual bool CompareScreenshots(
```

Comment:
I wonder if it would make more sense for this function to be together with the functions in FrameCapturePathManagementRequestBus. CompareScreenshots is really only relevant to automated testing, not capturing screenshots.

Reply:
If the function gets moved then should FrameCapturePathManagementRequestBus be renamed to account for everything it does beyond path management?